web/components/routes/page/detail/wrapper.tsx

```diff
@@ -105,20 +102,22 @@ const DetailPageForm = ({ page }: { page: PersonalPage }) => {
 	const handleContentKeyDown = useCallback((view: EditorView, event: KeyboardEvent) => {
 		const editor = contentEditorRef.current?.editor
 		if (!editor) return false
-
 		const { state } = editor
 		const { selection } = state
 		const { $anchor } = selection
-
-		if ((event.key === "ArrowLeft" || event.key === "ArrowUp") && $anchor.pos - 1 === 0) {
```

Comment:
@olyamrshn is there a reason why you removing this code? because users should be able to navigate between content to title with only keyboard